### PR TITLE
Prevent tour from displaying twice

### LINKED ISSUE #10196: User interaction submenu on Party modal Invites tab does nothing
[//]: # (Before logging this issue, please post to the Report a Bug guild from the Habitica website's Help menu. Most bugs can be handled quickly there. If a GitHub issue is needed, you will be advised of that by a moderator or staff member -- a player with a dark blue or purple name. It is recommended that you don't create a new issue unless advised to.)

[//]: # (Bugs in the mobile apps can also be reported there.)

[//]: # (If you have a feature request, use "Help > Request a Feature", not GitHub or the Report a Bug guild.)

[//]: # (For more guidelines see https://github.com/HabitRPG/habitica/issues/2760)

[//]: # (Fill out relevant information - UUID is found from the Habitia website at User Icon > Settings > API)
### Description
[//]: # (Describe bug in detail here. Include screenshots if helpful.)

Under at least some circumstances (the issue doesn't reproduce 100% of the time, though it's not clear why not), the three-dots menu button next to a user in the "Invites" tab of the party members modal doesn't do anything.

To reproduce:

1. Invite someone to your party.
2. Go to View Party, then Invites.
3. Click the "..." button at the right. If you hit this issue, nothing will happen. Compare to clicking the same button under the Members tab, which lets you send a message to the party member.

You should at least be able to rescind the party invitation from here, and preferably also have the same option of sending a private message to the user.

#### Console Errors
[//]: # (Include any JavaScript console errors here.)

None relevant.

### LINKED ISSUE #10414: Remove Member option doesn't appear when clicking View Party from main tasks page
[//]: # (Before logging this issue, please post to the Report a Bug guild from the Habitica website's Help menu. Most bugs can be handled quickly there. If a GitHub issue is needed, you will be advised of that by a moderator or staff member -- a player with a dark blue or purple name. It is recommended that you don't create a new issue unless advised to.)

[//]: # (Bugs in the mobile apps can also be reported there.)

[//]: # (If you have a feature request, use "Help > Request a Feature", not GitHub or the Report a Bug guild.)

[//]: # (For more guidelines see https://github.com/HabitRPG/habitica/issues/2760)

[//]: # (Fill out relevant information - UUID is found from the Habitia website at User Icon > Settings > API)
### Description
[//]: # (Describe bug in detail here. Include screenshots if helpful.)

When viewing a party's member list from the Party page proper, the party leader has the option to remove party members. This option is not, however, available from the View Party button on the main Tasks page, despite other options like Send Message and Promote to Leader showing up OK there.

### LINKED ISSUE #10509: Guild / Party Invite members button does not work for some players
Some players are reporting that the button to invite new members to a guild does not work for them.

I've seen several reports of this now so it's either a bug that affects only some OSes / user accounts or is an intermittent bug that's not platform-dependent or user-dependent. It does not occur for me. Some players report it happening in multiple browsers.

It seems to affect more than one guild for each player.

Here's one example and I'll post others soon:

----

Manifold (4bd2e5b3-7e24-43a8-9087-ced67ef6c9cb): "For the last week or so the invite button in any Guild I visit does not work when clicked. The button just shows the click animation but the pop over does not show allowing me to invite another member. I have tried Firefox and Chrome as well as Chrome Incognito to eliminate any extensions. Please let me know if you require further information."
 
Alys: "Please open the Developer Console or JavaScript Console in your browser (instructions here then if you can see the trash can icon in the Console, click that (it's probably at the top left of console but don't worry about this step if you don't see it). Then click on the invite button again. If anything appears in the Console, please paste it here. Don't worry about neat formatting for it."

Manifold: "Thanks for your reply. Nothing appears in the console when I click on the Invite button."

Alys: "I'm sorry we haven't got back to you before now. Please visit any guild you're a member of (this one will do if you've joined it), then after the guild page has fully loaded, reload the website using your brwoser's refresh button. Wait for it to fully load again, then click the invite button. Does that help? If it doesn't, please go to https://habitica.com/static/clear-browser-data and click the red "Clear Data" button there, then reload the website using your browser's refresh button while you are holding down the "Control" or "Command" key on your keyboard. Log in again, navigate back to any guild you're a member of, and try to invite again."

Manifold: "Thanks for getting back to me! :) I tried everything you suggested but the Invite dialog is still not opening. **I can use the API to invite members still.** Please let me know if I can provide you with any logs or other information to help debug this issue."

_I've asked them what OS they're running._

----

### PR DESCRIPTION
**Pull request by TheHollidayInn**
Fixes #10509
Fixes #10196

This may fix other issues. It affects a few items for new users.

### CODE REVIEW

File: website/client/mixins/guide.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import times from 'lodash/times';
-import Intro from 'intro.js/';
+import introjs from 'intro.js';
 import * as Analytics from 'client/libs/analytics';
 
+let showingTour = false;
```
Comment (paglias) [inline website/client/mixins/guide.js:5]: lgtm but this var shouldn't go inside  the component's data?

Comment (TheHollidayInn) [reply website/client/mixins/guide.js:5]: Nah, it needs to be shared across components like a singleton. Adding it to the components data will allow it to be reset when a component is mounted. I could put it in the Store, but that seems like overkill.

**Comment by paglias** [comment]
schermata 2018-08-17 alle 12 22 35

Not sure if it's related but I'm still getting these errors on new accounts

**Comment by TheHollidayInn** [comment]
This fixes that issue. That looks like you aren't using code with this PR because I've updated the warning about using the correct introjs method.

**Comment by paglias** [comment]
Sounds good, I thought I was using the up to date version but probably got confused

**Pull request merged**